File: 백준/Bronze/25640. MBTI/MBTI.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String jinho = sc.nextLine();
        int n = Integer.parseInt(sc.nextLine());

        ArrayList<String> mbti = new ArrayList<>();
        for (int i = 0; i < n; i++) {
            mbti.add(sc.nextLine());
        }

        long count = mbti.stream().filter(m -> m.equals(jinho)).count();
        System.out.println(count);
    }
}

File: 백준/Bronze/25640. MBTI/MBTI.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        string jinho = Console.ReadLine();
        int n = int.Parse(Console.ReadLine());
        
        List<string> mbti = new List<string>();
        for (int i = 0; i < n; i++)
        {
            mbti.Add(Console.ReadLine());
        }

        Console.WriteLine(mbti.FindAll(x => x == jinho).Count);
    }
}
